docs/reference/installation.rst

```diff
@@ -70,10 +67,6 @@ Add these bundles in the config mapping definition (or enable `auto_mapping`_):
                         ApplicationSonataPageBundle: ~ # only once the ApplicationSonataPageBundle is generated
                         SonataPageBundle: ~
 
-        dbal:
-            types:
-                json: Sonata\Doctrine\Types\JsonType
```

Comment:
[JsonType is deprecated ](https://github.com/sonata-project/sonata-doctrine-extensions/blob/1.x/src/Types/JsonType.php)in favor of the Doctrine one.